### PR TITLE
Dashboard Protype

### LINKED ISSUE #10: Firebase SDK Dashboard Common Styling.

### PR DESCRIPTION
**Pull request by TMazenge**
Implemented the style on the landing page and moved it to an index.html file. Added SDK list page.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>com.google.sps</groupId>
+  <groupId>com.google.capstone</groupId>
   <artifactId>sdktiktok</artifactId>
```
Comment (nnenna352) [inline pom.xml]: I think the name should be changed from sdktiktok since we didn't agree on a name yet

File: src/main/webapp/index.html [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    <script src="js/script.js"></script>
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js" integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  </body>
+</html>
+
+
```
Comment (nnenna352) [inline src/main/webapp/index.html]: end of file error

File: pom.xml (hunk 1 above)
Comment (TMazenge) [reply pom.xml]: Yes I will change it when we decide upon the name.

File: pom.xml [hunk 3]
```diff
@@ -3,7 +3,7 @@
     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>com.google.sps</groupId>
+  <groupId>com.google.capstone</groupId>
```
Comment (ankitaj224) [inline pom.xml]: Have a name specific to dashboard. Refer to #18 package name

File: src/main/webapp/release_history.html [hunk 4]
```diff
@@ -16,30 +16,52 @@ <h2>Firebase Common</h2>
...
+      <h4>Owner: ashwinraghav@</h4>
+      <div id="release-history" class="table-responsive">
+        <table class="table">
+          <caption>Version History</caption>
+          <thead>
+            <tr>
+              <th scope="col">Version</th>
+              <th scope="col">Launch Date</th>
+              <th scope="col">Branch</th>
+              <th scope="col">Minor/Major Bump</th>
```
Comment (ankitaj224) [inline src/main/webapp/release_history.html]: You can remove this column

File: src/main/webapp/release_sdks.html [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+                <div class="sdk">
+                    Firebase Messaging       
+                </div>
+           </div>
+        </div>
+
+         <div class="col" >
+            <div class="row justify-content-center">
+                <div class="sdk">
+                    Firebase IDD
```
Comment (ankitaj224) [inline src/main/webapp/release_sdks.html]: nit : IID

**Review by ankitaj224 — APPROVED** [review]
See, if you can avoid submitting files under target/

**Pull request merged**